### PR TITLE
Y2021 ex7

### PR DESCRIPTION
**Pull request by lmammino**
Solves p1 using median and p2 using avg and gauss

### CODE REVIEW

File: y2021/ex07/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+pub fn part1(input: &str) -> i32 {
+    let mut positions: Vec<i32> = input
+        .split(',')
+        .map(|n| n.parse::<i32>().unwrap())
+        .collect();
+    positions.sort_unstable();
```
[the remark this answers is not in the document]
Comment (pepoviola) [reply y2021/ex07/src/lib.rs:6]: Hi @lmammino, I was watching Yosh streaming today (https://www.twitch.tv/yoshuawuyts) and he use this nice way to get the `median` from unsorted.

```rust
    let nth = positions.len() / 2;
    let (_, median, _) = nums.select_nth_unstable(nth);
```

[select_nth_unstable doc](https://doc.rust-lang.org/beta/std/primitive.slice.html#method.select_nth_unstable)

👋

[the remark this answers is not in the document]
Comment (lmammino) [reply y2021/ex07/src/lib.rs:6]: Wow, this is super cool! Thanks for sharing this tip :)

[the remark this answers is not in the document]
Comment (pepoviola) [reply y2021/ex07/src/lib.rs:6]: Yes!, super nice method 🚀

**Comment by lmammino** [comment]
Great job @gambuzzi + @malagolichiara

**Pull request merged**